### ISSUE TITLE
trouble with fetching the logs

### ISSUE THREAD

**Issue by Unimax** [openai/coinrun#28]
I am trying to recreate the figure 6 in the paper which show the effect of all the known techniques on performance for coinrun. Need help with the following if possible.

- starting new training just clear the directory and delete all previous logs!!

- the train with --test seems not to produce enough information on tensorboard. rew_mean_<run_id> is the mean reward for testing on random levels I suppose. training mean reward results are missing?

![image](https://user-images.githubusercontent.com/4117562/54728305-489dd380-4b3a-11e9-9dc6-a24056c712a2.png)

- resuming the training starts the episodes and total timestamps from 0 shouldn't it start from the previous numbers. (i can see the model is loaded and getting improved). its ends up with wrong graphs in the tensorboard.

- what changes come into mind if I try to apply DQN here?

Thanks in advance

**Comment by sanjayyyyyy** [comment]
How do we fetch the logs after testing

**Comment by Unimax** [comment]
default dir is '/tmp/tensorflow'
use tensorboard to plot the logs
note: by default, it will remove all previous logs. idk why they wanted that.
